tests/clients/layer0_test_client.go

```diff
@@ -23,93 +21,169 @@ func NewLayer0TestClient(t *testing.T, endpoint, token string) *Layer0TestClient
 	}
 }
 
-func (l *Layer0TestClient) jobHelper(fn func() (string, error)) string {
-	jobID, err := fn()
+func (l *Layer0TestClient) CreateTask(taskName, environmentID, deployID string, copies int, overrides []models.ContainerOverride) string {
```

Comment:
This is not the established pattern: we pass in request models now. The test client has already been updated to reflect this; any additional functionality should adhere to those patterns.